web/src/lib/components/admin-page/settings/machine-learning-settings/machine-learning-settings.svelte

```diff
@@ -77,6 +77,40 @@
         </div>
       </SettingAccordion>
 
+      <SettingAccordion
+        key="duplicate-detection"
+        title="Duplicate Detection"
+        subtitle="Use CLIP embeddings to find very similar images that are likely duplicates"
```

Comment:
```suggestion
        subtitle="Use CLIP embeddings to find likely duplicates"
```
This is a little wordy.